cprover-cpp-api/readme.md

```diff
@@ -0,0 +1,18 @@
+# The CPROVER C++ API
+
+This folder includes the interface and the implementation of a new C++-based
+API for the CProver libraries. The aim is for this to be able to be used by
+clients to drive all of the verification pipeline without the need to invoke
+any of the associated tools.
```

Comment:
This [folder/directory] includes the interface and implementation of a C++-based API for the CProver libraries. This allows direct linking to key CProver functionality via C++ function calls. This (will eventually) allow the use of all aspects of the CProver verification pipeline via direct function calls and deprecate the need for invoking separate tools/programs.